control-plane/.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+package-lock.json
```

Comment:
This file belongs in the repo, please remove from .gitignore
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
Done